resources/quality/ankermake_m5c/ankermake_m5c_fast.inst.cfg

```diff
@@ -0,0 +1,87 @@
+[general]
+definition = ankermake_m5c
+name = Fast
+version = 4
+
+[metadata]
+global_quality = True
+quality_type = fast
+setting_version = 20
+type = quality
+weight = -2
+
+[values]
+layer_height = 0.25
+layer_height_0 = 0.14
+
+speed_travel=500
+speed_print=500
+speed_infill=270
+speed_wall=250
+speed_wall_0=150
+speed_wall_x=250
+speed_topbottom=150
+speed_roofing=150
+speed_support=250
+speed_support_infill=250
+speed_support_interface=166.667
+speed_support_roof=166.667
+speed_support_bottom=166.667
+speed_prime_tower=500
+speed_layer_0=50
+speed_print_layer_0=50
+speed_travel_layer_0=150
+skirt_brim_speed=50
+speed_z_hop=10
+speed_slowdown_layers=2
+speed_equalize_flow_enabled=false
+speed_equalize_flow_max=300
+acceleration_enabled=true
+acceleration_travel=5000
+acceleration_print=5000
+acceleration_infill=5000
+acceleration_wall=5000
+acceleration_wall_0=3000
+acceleration_wall_x=5000
+acceleration_topbottom=2500
+acceleration_roofing=2500
+acceleration_support=5000
+acceleration_support_infill=5000
+acceleration_support_interface=5000
+acceleration_support_roof=5000
+acceleration_support_bottom=5000
+acceleration_prime_tower=5000
+acceleration_layer_0=2500
+acceleration_travel_layer_0=2500
+acceleration_print_layer_0=2500
+acceleration_skirt_brim=2500
+
+jerk_enabled=true
+jerk_travel=15
+jerk_print=15
+jerk_infill=15
+jerk_wall=15
+jerk_wall_0=15
+jerk_wall_x=15
+jerk_topbottom=15
+jerk_roofing=15
+jerk_support=15
+jerk_support_infill=15
+jerk_support_interface=15
+jerk_support_roof=15
+jerk_support_bottom=15
+jerk_prime_tower=15
+jerk_layer_0=15
+jerk_travel_layer_0=15
+jerk_print_layer_0=15
+jerk_skirt_brim=15
+ak_VAJK_J_E_enabled=true
+ak_J_E_print=3
+ak_J_E_infill=3
+ak_J_E_wall=3
+ak_J_E_wall_0=3
+ak_J_E_wall_x=3
+ak_J_E_topbottom=3
+ak_J_E_support=3
+ak_J_E_skirt_brim=3
```

Comment:
What are those settings (the ones with ak_J_E in front of them)? They arent settings that exist in Cura.

Reply:
Thank you for pointing that out.  I will remove them from the configuration file